dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -697,13 +708,15 @@ CONFIG_HAVE_KVM_NO_POLL=y
 CONFIG_KVM_XFER_TO_GUEST_WORK=y
 CONFIG_HAVE_KVM_PM_NOTIFIER=y
 CONFIG_KVM_GENERIC_HARDWARE_ENABLING=y
+CONFIG_KVM_GENERIC_MMU_NOTIFIER=y
 CONFIG_VIRTUALIZATION=y
 CONFIG_KVM=y
-# CONFIG_KVM_WERROR is not set
+# CONFIG_KVM_SW_PROTECTED_VM is not set
```

Comment:
This may be interesting to enable.
It's in KVM and it looks reachable w/o any special setup. May have some new APIs.